internal/wire/parse.go

```diff
@@ -1238,3 +1199,21 @@ func (pt ProvidedType) Field() *Field {
 	}
 	return pt.f
 }
+
+// loadWireLib loads the wire package the user is importing from their injector.
+// The call is a wire marker function call.
+func loadWireLib(info *types.Info, call *ast.CallExpr) *types.Package {
```

Comment:
Nit: instead of extracting just the package type-checking info and making the caller do the lookup, let's make this function directly obtain the `bindToUsePointer` and return a bool. (This would naturally come with a name change to reflect purpose, perhaps `shouldBindUsePointer`.)

Reply:
Done.